File: playground/PracticeEnum.java
package com.practice.enums;

enum Weekday {
    MONDAY,
    TUESDAY,
    WEDNESDAY,
    THURSDAY,
    FRIDAY,
    SATURDAY,
    SUNDAY;

    public String toString() {
        return name().toLowerCase();
    }
}

class PracticeEnum {

    public static void main(String[] args) {
        Weekday[] days = Weekday.values();
        for (Weekday day : days) {
            System.out.println(day);
        }
    }
}

File: playground/PracticeEnum.scala
package com.scalapractice.enums

enum Weekday:
    case Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday

@main def main(args: String*): Unit =
    val weekday = Array(
        Weekday.Monday, Weekday.Tuesday, Weekday.Wednesday,
        Weekday.Thursday, Weekday.Friday, Weekday.Saturday, Weekday.Sunday
    )
    println(weekday.mkString(", "))
